sentry-ruby/lib/sentry/event.rb

```diff
@@ -18,7 +18,7 @@ class Event
       event_id level timestamp
       release environment server_name modules
       message user tags contexts extra
-      fingerprint breadcrumbs transaction
```

Comment:
It looks like a `TransactionEvent`-only attribute? If that's the case, can we put it there?

Reply:
I had it there first, but then moved it here because it's always coupled to the `transaction` field.